libs/qwikdev-astro/server.ts

```diff
@@ -60,9 +62,10 @@ export async function renderToStaticMarkup(
       }
     }
 
-    const app = jsx(Component, {
+    const slotValues = Object.values(slotted);
+    const app = jsx(component, {
       ...props,
-      children: [defaultSlot, ...Object.values(slots)],
+      children: defaultSlot ? [defaultSlot, ...slotValues] : [...slotValues]
```

Comment:
```suggestion
      children: defaultSlot ? [defaultSlot, ...slotValues] : [...slotValues]
```

On first glance this should work. I am hoping at some point to setup playwright tests where we can have a test suite for changes like this.